storey/flow.py

```diff
@@ -629,3 +664,150 @@ def build_flow(steps):
+                body = await response.text()
+                raise V3ioError(f'Failed to save schema file. Response status code was {response.status}: {body}')
+        except BaseException as ex:
+            raise ex
+        finally:
+            await client_session.close()
+
+    async def load_schema(self):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
+
+        try:
+            schema_path = f'{self._webapi_url}/{self.table}/{schema_file_name}'
+            response = await client_session.get(schema_path, headers=self._get_put_file_headers, ssl=False)
+            body = await response.text()
+            if response.status == 404:
+                schema = None
+            elif response.status == 200:
+                schema = json.loads(body)
+            else:
+                raise V3ioError(f'Failed to get schema at {schema_path}. Response status code was {response.status}: {body}')
+        except BaseException as ex:
+            raise ex
+        finally:
+            await client_session.close()
+
+        return schema
+
+    async def save_store(self, aggr_store):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
+
+        for key, aggregation_element in aggr_store.items():
+            data = {'Item': self._get_attributes_as_blob(aggregation_element), 'UpdateMode': 'CreateOrReplaceAttributes'}
+            response = await client_session.put(f'{self._webapi_url}/{self.table}/{key}',
+                                                headers=self._put_item_headers, data=json.dumps(data), ssl=False)
+            if not response.status == 200:
+                body = await response.text()
+                raise V3ioError(f'Failed to save aggregation for key: {key}. Response status code was {response.status}: {body}')
+
+        await client_session.close()
+
+    async def save_key(self, key, aggr_item):
+        connector = aiohttp.TCPConnector()
+        client_session = aiohttp.ClientSession(connector=connector)
+
+        data = {'Item': self._get_attributes_as_blob(aggr_item), 'UpdateMode': 'CreateOrReplaceAttributes'}
+        response = await client_session.put(f'{self._webapi_url}/{self.table}/{key}',
+                                            headers=self._put_item_headers, data=json.dumps(data), ssl=False)
+        if not response.status == 200:
+            body = await response.text()
+            raise V3ioError(f'Failed to save aggregation for key: {key}. Response status code was {response.status}: {body}')
+
+        await client_session.close()
+
+    def _get_attributes_as_blob(self, aggregation_element):
+        data = {}
+        for name, bucket in aggregation_element.aggregation_buckets.items():
+            # Only save raw aggregates, not virtual
+            if bucket.should_persist:
+                blob = pickle.dumps(bucket.to_dict())
```

Comment:
Pickling a dict has space and processing overheads compared to exporting a byte array. Maybe we can leave this one for later, though.